app/src/ui/merge-branch/merge.tsx

```diff
@@ -230,6 +231,12 @@ export class Merge extends React.Component<IMergeProps, IMergeState> {
     return renderDefaultBranch(item, matches, this.props.currentBranch)
   }
 
+  private onDismissed = () => {
+    this.state.filterText.length > 0
+      ? this.setState({ filterText: '' })
```

Comment:
I think eslint here is not a fan of this coding style. Switching it back to a plain `if` should make it happy:

```ts
if (this.state.filterText.length > 0) {
  this.setState({ filterText: '' })
} else {
  this.props.onDismissed()
}
```